### ISSUE TITLE
Wallet "Sweep" Error...?

### ISSUE THREAD

**Issue by KeyJockey** [Bitcoin-com/Wallet#56]
Hey guys... I'm cross posting this here from a post on bitcoin.com forum for the copay-forked wallet version JUST because I'm kind of in a small panic and hopefully someone here will be able to help/reassure me a little faster, LOL.

Anyway... cross-post text from there, here below.

Thanks in advance for any/all replies.

---

I'm having a problem with a transaction I made using the latest version of the bitcoin.com wallet.

"Sweeping" a Bitcoin Cash "BCH" balance from a private key at first, the transaction appeared to work and it's showing in my wallet, but says unconfirmed.

I have "use unconfirmed funds" enabled so I went ahead and did another transaction to SEND some BCH to another address.

That transaction also appeared to work, and it's in the wallet also showing unconfirmed.

HOWEVER there is nothing appearing on the blockchain for either transaction... the original "swept" address still shows the full balance (i.e. "unswept" still... untouched) and with only ONE transaction on that address (i.e. the initial "deposit" for that address, cold wallet).

Any search for the transaction from the bitcoin.com wallet yields "transaction not found"... and it did not "time out" but rather it was NEVER able to be found, even a minute or two after the transaction completed.

It kind of appears like the wallet did not "broadcast" anything... but it THINKS it did...?

I also tried also to use another wallet to sweep the original address again... but that gave an error message (somewhat frighteningly) about something like nothing available to sweep or no inputs or something like that.... even though on several block explorers the full original balance IS showing there, still at the original address with the same original balance amount.

Hopefully someone knows what's going on here... ??  and I hope I have not lost these funds (it's kind of a large balance so I'm worried...)

Thanks in advance for all advice how to resolve this and get access to my BCH again.....

**Comment by propha** [comment]
Im having a lot of prob with everything

**Comment by KeyJockey** [comment]
(Cross-Post, again, from bitcoin.com forum) 
=====
Just an update here: So far I have heard nothing from any "official" support channel for the bitcoin.com wallet... and it's looking to me more and more like this IS some kind of serious bug in the latest version of your wallet.

Jonald Fyookball (dev of the Electron Cash wallet) has been helping me over on reddit (from my post there about this) via PM message... and so far we've been able to access the problem transaction via importing my privkey from the address that was "swept" by the bitcoin.com wallet.

And there IS definitely something very wrong with that transaction. Somehow -- not sure yet exactly what happened -- the bitcoin.com wallet "sweep" function has created an apparently malformed transaction that "does not exist" in any block explorer, and that has apparently "sent" my bitcoins to an address where they are now definitely NOT residing, leaving them actually still "stuck" in the ORIGINAL address but yet in-accessible to any other wallet.

Bottom line? User-Error or not (I don't think so, all I did was import/sweep a priv key in the bitcoin.com wallet as is supposed to be able to be done) this is looking like a very serious GLITCH or BUG in the bitcoin.com wallet that has at least temporarily -- but hopefully not permanently -- locked my funds away from my control.

I'd really like to get this issue "escalated" some way, if that's possible... i.e. bring it to attention of the actual bitcoin.com wallet developers to get it resolved and not have to rely only on the kindness of support of other wallet devs like Jonald........???

**Comment by KeyJockey** [comment]
UPDATE: wow, Jonald Fyookball kicked ass, "Fyookball FTW" totally!!!

Helping me via PM on Reddit, Jonald got my stuck funds OUT from this clearly gakked-up transaction... no idea yet really what happened BUT seems to me that the bitcoin.com wallet *DOES* have some kind of SERIOUS problem with the "sweep" function.

Here's what Jonald told me to do, that got my coins back:

---

"What is happening is that either your local wallet or the electrum server you're connecting to believes there is a transaction, but other network nodes do not.

So, don't panic because the network simply doesn't see any transaction.
We just have to get rid of the problem locally.

If I load up your address in a fresh wallet, it doesn't show the unconfirmed transaction.

Try this:

Delete the wallet file you have now that is watching it.
Delete the recent_servers file delete the blockchain_headers file
Run this command from a cmd prompt: this will force it run from a single server:

ElectronCash-3.1.2.exe -1 -s electroncash.checksum0.com:50002:s

Now create the wallet again using the private key.
Don't sweep, just create the imported private key wallet.

---

I did that, and the original BCH address that I'd tried to sweep-from appeared in the new wallet with NO SIGN of the bad transaction any more!

I was able to then send the coins again to a new, safe, clean address... problem solved!

Again, thanks a million to Jonald Fyookball (I tipped him, of course, but FWIW anyone else reading this today who has some spare BitcoinCash laying around by all means I nominate Jonald as MAN OF THE HOUR today for tips... he's a credit to our community, and a gentleman and a scholar indeed) 😎
